src/popover/internal.tsx

```diff
@@ -111,7 +111,7 @@ function InternalPopover(
   };
 
   const popoverContent = (
-    <span
+    <div
```

Comment:
This change is in fact breaking, because there are use-cases when popover is rendered in the inline context. Check the screenshot tests results for the detailed report